common/expression/src/display.rs

```diff
@@ -277,3 +262,81 @@ impl Debug for FunctionProperty {
         write!(f, "{self}")
     }
 }
+
+impl Display for NullableDomain<AnyType> {
+    fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
+        if let Some(value) = &self.value {
+            if self.contains_null {
+                write!(f, "{} ∪ {{NULL}}", value)
```

Comment:
Is using `|` instead of `∪` a better idea?

For example:

Before: `({FALSE}, {"a"..="e"}) ∪ {NULL}`
After: `({FALSE}, {"a"..="e"}) | {NULL}`

Reply:
I chose `∪` because it means the union of two sets. `|` seems a little be vague in sematic to me, because `{FALSE, TRUE} | {NULL}` sounds like `oh, it may be ALL BOOLEAN or ALL NULL in the datablock, but anyway you can probe it` instead of `it could be an only could be any scalar in {TRUE, FALSE, NULL}`.

Reply:
LGTM